src/language/de.rs

```diff
@@ -0,0 +1,18 @@
+use super::*;
+
+use Error::*;
+
+impl Language {
+    pub fn de_message(error: Error) -> &'static str {
+        match error {
+            UnsafePasswordTooShort => "Das Passwort muss mindestens 8 Buchstaben lang sein.",
```

Comment:
Shouldn't this be "Zeichen"? "Buchstaben" implies alphabetic characters.

Reply:
ah yes, good idea. I will change it :)